hack/deployer/runner/eks.go

```diff
@@ -39,15 +39,10 @@ metadata:
     {{- end }}
 nodeGroups:
   - name: ng-1
-    amiFamily: AmazonLinux2
+    amiFamily: AmazonLinux2023
     instanceType: {{.MachineType}}
     desiredCapacity: {{.NodeCount}}
     ami: {{.NodeAMI}}
-    overrideBootstrapCommand: |
-      #!/bin/bash
-      source /var/lib/cloud/scripts/eksctl/bootstrap.helper.sh
-
-      /etc/eks/bootstrap.sh {{.ClusterName}} --container-runtime containerd --kubelet-extra-args "--node-labels=${NODE_LABELS}"
```

Comment:
This is no longer supported on `AmazonLinux2023`, and does not seem to be required 🤷